Amplify/Categories/Hub/HubFilter.swift

```diff
@@ -31,8 +31,9 @@ public struct HubFilters {
 
     /// Returns a HubFilter that is `true` if the event's `context` property has a UUID that matches `operation.id`
     /// - Parameter operation: The operation to match
-    public static func forOperation<Request: AmplifyOperationRequest, Success, Failure: AmplifyError>
-        (_ operation: AmplifyOperation<Request, Success, Failure>) -> HubFilter {
+    public static func forOperation<Request: AmplifyOperationRequest>
+        (_ operation: AmplifyOperation<Request, some Any, some AmplifyError>) -> HubFilter
+    {
```

Comment:
The formatting here seems messed up. It should be

```swift
    public static func forOperation<Request: AmplifyOperationRequest>(
        _ operation: AmplifyOperation<Request, some Any, some AmplifyError>
    ) -> HubFilter {
```

Reply:
moved by to same line.